imagemanager/imagemanager.go

```diff
@@ -25,13 +25,15 @@ import (
 	"net/url"
 	"os"
 	"path"
+	"slices"
```

Comment:
will it compile on Yocto?
As I remember we have older go version there and use experimental package
"golang.org/x/exp/slices"